src/adapters/model_mixin.py

```diff
@@ -1617,6 +1621,70 @@ def save_pretrained(
         # Remove adapters config
         del self.config.adapters
 
+    def gradient_checkpointing_enable(self, gradient_checkpointing_kwargs=None):
```

Comment:
Can we add a small note from where this method is overriden?

Reply:
Yes, will do this :+1: